style.css

```diff
@@ -0,0 +1,1498 @@
+  background-blend-mode: overlay;
+  background-position: top;
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+
+.card-3-content-5 {
+  margin-top: 16px;
+  padding: 12px;
+  height: 276px;
+  width: 100%;
+  background-image: url("https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2020/07/a-female-office-worker-at-the-table-holding-papers-with-diagrams-796x531.jpg"), linear-gradient(transparent, rgba(0, 0, 0) 98%);
+  background-blend-mode: overlay;
+  background-position: top;
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+
+.card-3-content-6 {
+  margin-top: 16px;
+  padding: 12px;
+  height: 276px;
+  width: 100%;
+  background-image: url("https://img-cdn.tnwcdn.com/image?fit=1280%2C720&url=https%3A%2F%2Fcdn0.tnwcdn.com%2Fwp-content%2Fblogs.dir%2F1%2Ffiles%2F2020%2F07%2F106909103_10101777330197904_2889321509499508446_o-1.jpg&signature=c9e036a88da7384ae8e70804f7af81e0"), linear-gradient(transparent, rgba(0, 0, 0) 98%);
+  background-blend-mode: overlay;
+  background-position: top;
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+
+.card-3-content-7 {
+  margin-top: 16px;
+  padding: 12px;
+  height: 276px;
+  width: 100%;
+  background-image: url("https://img-cdn.tnwcdn.com/image?fit=1280%2C720&url=https%3A%2F%2Fcdn0.tnwcdn.com%2Fwp-content%2Fblogs.dir%2F1%2Ffiles%2F2020%2F08%2Fastronautpoop.jpg&signature=dd00926d0ddc41a4e4dffbe6f9b4775a"), linear-gradient(transparent, rgba(0, 0, 0) 98%);
+  background-blend-mode: overlay;
+  background-position: top;
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+
+.card-3-content-8 {
+  margin-top: 16px;
+  padding: 12px;
+  height: 276px;
+  width: 100%;
+  background-image: url("https://img-cdn.tnwcdn.com/image?fit=1280%2C720&url=https%3A%2F%2Fcdn0.tnwcdn.com%2Fwp-content%2Fblogs.dir%2F1%2Ffiles%2F2020%2F07%2FShift-launch-cars-evs-future-polestar-electric-cars-driving-mobility.png&signature=ae628edcb255606b8d38e0e7a9a34998"), linear-gradient(transparent, rgba(0, 0, 0) 98%);
+  background-blend-mode: overlay;
+  background-position: top;
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+
+.card-3-content-9 {
+  margin-top: 16px;
+  padding: 12px;
+  height: 276px;
+  width: 100%;
+  background-image: url("https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2020/08/Mulan-796x417.jpg"), linear-gradient(transparent, rgba(0, 0, 0) 98%);
+  background-blend-mode: overlay;
+  background-position: top;
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+
+.card-3-bottom {
+  background-color: rgba(255, 68, 34, 0.08);
+  color: #f42;
+  display: flex;
+  padding-left: 14px;
+}
+
+.card-3-bottom :first-child {
+  padding: 7px 0 1px;
+}
+
+.card-4-discount h4 {
+  transform: rotate(45deg);
+  font-size: 14px;
+  position: absolute;
+  top: 20px;
+  right: 0;
+}
+
+.card-3-bottom div h4 {
+  font-family: "ARS Maquette-bold", serif;
+  font-size: 14px;
+}
+
```

Comment:
## Using Inefficient CSS Selectors[1]
When you are using selectors, the longer it is, the higher specificity it will comprise. This can break the CSS cascade and mess up the styles of your site.

This can be avoided by not using IDs within the selectors as they are directly specific.

 - [ ]  Please add a class to `<a>` element` and delete the left unnecessary rules

**Example:**
```CSS
/* before */
.card-3-bottom div h4 {
/* after */
.h4-class {
```
<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_